plugin.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+	"github.com/mattermost/mattermost-server/plugin/rpcplugin"
+)
+
+type MatterpollPlugin struct{}
+
+func (p *MatterpollPlugin) OnActivate(api plugin.API) error {
+	return api.RegisterCommand(&model.Command{
+		DisplayName:      `Matterpoll`,
+		Trigger:          `matterpoll`,
+		AutoComplete:     true,
+		AutoCompleteDesc: `Create a poll`,
+		AutoCompleteHint: `[Question] [Option1] [Option2]...`,
```

Comment:
`[OptionX]` is called `Answer 1` in line 29.
Ithink that it is better to be called same name.

I like `answer` because `Option` looks little similar to `Question`.

Reply:
Your right :+1: